RadarSDK/remodel/RadarBeacon.h

```diff
@@ -0,0 +1,65 @@
+/**
+ * This file is generated using the remodel generation script.
+ * The name of the input file is RadarBeacon.value
+ */
+
+#import "RadarJSONCoding.h"
+#import "RadarLocation.h"
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * This is a test data class for auto generation code
+ */
+@interface RadarBeacon : NSObject<RadarJSONCoding, NSCopying, NSCoding>
```

Comment:
know this is autogenerated, but seems redundant to add annotations for `NSCopying` / `NSCoding` if anything that abides by `RadarJSONCoding` will necessarily have those as well? is it worth putting those annotations on `RadarJSONCoding`? there might be some nuance between `@interface` and `@protocol` that I don't understand